### PR TITLE
Detached nymph switches to gestalt upon death

### LINKED ISSUE #6780: Dionae Gestalt Detatched Nymph Death
Hi! I am aware that a Gestalts ability to detach a limb such as an arm to become a controllable nymph is a relatively new addition to the mechanics. But i have discovered an issue. Whilst playing as said nymph, If the nymph is killed you are sent into ghost mode. Your character is completely dead and you are unable to just die as a nymph and return to your LIVING gestalt body. 

Note: I have only died as a nymph whilst crawling through the vents on station and being sucked into a valve or scrubber. But im pretty sure the bug applies to any nymph death.

### PR DESCRIPTION
**Pull request by Sindorman**
- Detached nymph now switches to gestalt upon death. Fixes #6780

### CODE REVIEW

**Comment by Sindorman** [comment]
!review

File: code/modules/mob/living/carbon/diona_base.dm [hunk 1]
```diff
@@ -58,6 +58,11 @@ var/list/diona_banned_languages = list(
 		sprint_speed_factor = 0.75
 		sprint_cost_factor = 0
 
+/mob/living/carbon/alien/diona/death()
+	if(client)
+		to_chat(switch_to_gestalt(), span("warning", "You feel tremendous pain as you loose connection to your detached nymph, there is no way to bring it back anymore."))
```
Comment (skull132) [inline code/modules/mob/living/carbon/diona_base.dm]: `switch_to_gestalt()` doesn't return any valid values.

```dm
/mob/living/carbon/alien/diona/proc/switch_to_gestalt()
	set name = "Switch to Gestalt"
	set desc = "Allows you to switch control back to your parent Gestalt."
	set category = "Abilities"

	if(!gestalt)
		to_chat(src, span("warning", "You have no Gestalt!"))
	else if(gestalt.stat == DEAD)
		to_chat(src, span("danger", "Your Gestlat is not responding! Something could have happened to it!"))
	else
		gestalt.key = key
```

Comment (Conspiir) [inline code/modules/mob/living/carbon/diona_base.dm]: Small typo
"Loose" should become "lose"

Comment (Conspiir) [inline code/modules/mob/living/carbon/diona_base.dm]: Small typo
"Loose" should become "lose"

Comment (Sindorman) [reply code/modules/mob/living/carbon/diona_base.dm]: Yes it does, line 594 of changed files. Your probably took this code from master

**Pull request merged**